### ISSUE TITLE
Error: Cannot find module 'react-i18next'  

### ISSUE THREAD

**Issue by brandongregoryscott** [AndcultureCode/AndcultureCode.JavaScript.React#14]
Looks like the same issue as before - this package should be moved to the production dependency list so it is bundled up in the build.

```
Uncaught Error: Cannot find module 'react-i18next'
    at webpackMissingModule (use-localization.ts:2)
    at Object../node_modules/andculturecode-javascript-react/dist/hooks/use-localization.js (use-localization.ts:2)
    at __webpack_require__ (bootstrap:784)
    at fn (bootstrap:150)
    at Object../node_modules/andculturecode-javascript-react/dist/index.js (index.ts:30)
    at __webpack_require__ (bootstrap:784)
```

**Comment by wintondeshong** [comment]
bah!!! one day I'll stop doing this :(

**Comment by wintondeshong** [comment]
Fixed in release 0.0.9
